AsyncDisplayKit/ASTableNode.h

```diff
@@ -287,6 +287,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (NSInteger)tableNode:(ASTableNode *)tableNode numberOfRowsInSection:(NSInteger)section;
 
 /**
+ * Similar to -visibleCells.
+ *
+ * @return an array containing the nodes being displayed on screen. This must be called on the main thread.
+ */
+- (NSArray<__kindof ASCellNode *> *)visibleNodes AS_WARN_UNUSED_RESULT;
```

Comment:
Interesting. UITableView declares `visibleNodes` as a property, `UICollectionView` declares it as a property. I suppose we should just mirror their declarations so that Swift migrations don't have a senseless removal of `()`.